src/PaySimpleSdk/Customers/CustomerService.cs

```diff
@@ -259,5 +259,21 @@ internal CustomerService(IPaySimpleSettings settings, IValidationService validat
             var result = await webServiceRequest.PutDeserializedAsync<Customer, Result<Customer>>(new Uri(endpoint), customer);
             return result.Response;
         }
+
+	    public async Task<PaymentToken> MatchOrCreateCustomerAndCreditCardAccountAsync(CustomerAndAccountRequest request)
+	    {
+			validationService.Validate(request);
+			var endpoint = $"{settings.BaseUrl}{Endpoints.MatchOrCreateCustomerAndCreditCardAccount}/";
```

Comment:
probably nit-picking on this one, but none of our other (existing) endpoints end with a / unless we have things to put after it, should follow convention here.